src/components/containers/ArticleLists/ArticleLists.jsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable react/jsx-key */
```

Comment:
let's remove this file since its not used used here.

Reply:
done, thanks.